<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogprogram_framemove</class>
 <widget class="QDialog" name="dialogprogram_framemove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>131</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="Edit_W">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Set">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit_V">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit_Z">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Move Frame ( mm )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Relative_Add">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Relative_Init">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit_Y">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit_X">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit_U">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Relative_Set">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Get">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>131</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Relative</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>Edit_W</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_Add</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton_Set</zorder>
  <zorder>Edit_V</zorder>
  <zorder>label_2</zorder>
  <zorder>Edit_Z</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_Relative_Add</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_Relative_Init</zorder>
  <zorder>Edit_Y</zorder>
  <zorder>Edit_X</zorder>
  <zorder>Edit_U</zorder>
  <zorder>pushButton_Relative_Set</zorder>
  <zorder>pushButton_Get</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
